pocean/utils.py

```diff
@@ -284,6 +286,7 @@ def create_ncvar_from_series(ncd, var_name, dimensions, series, **kwargs):
                 raise ValueError('datatype error: boolean needs to be converted to string')
         except BaseException:
             # Fall back to a string type
+            kwargs.update({"zlib": False})
```

Comment:
It turns out we cannot compress strings! This always failed but it was more forgiving in the past.

See https://github.com/Unidata/netcdf-c/pull/2231